ui/pages/AnvilWorkspace/components/LoadWorkspaceDataForm.jsx

```diff
@@ -81,7 +81,7 @@ const SAMPLE_TYPE_FIELD = {
 const DATA_BUCK_FIELD = {
   name: 'dataPath',
   label: 'Path to the Joint Called VCF',
-  labelHelp: 'File path for a joint called VCF available in the workspace "Files". If the VCF is split, provide the path to the directory containing the split VCF',
+  labelHelp: 'File path for a joint called VCF available in the AnVIL workspace\'s "Files" folder.',
```

Comment:
"Files" isn't a folder so adding that word makes no sense

Reply:
How about changing it to `File path for a joint called VCF available in the AnVIL workspace's storage bucket.`? Because `Files` itself doesn't make much sense even it is shown in the AnVIL UI.

Reply:
<img width="293" alt="Screen Shot 2021-08-09 at 4 39 12 PM" src="https://user-images.githubusercontent.com/24598672/128771314-397c179b-5f43-4cb1-808c-6d75777c73e9.png">

Its literally that pat of workspaces that is labeled "Files", and if the file is somewhere else in the workspace's storage it won't work. So no. Please do not change this to something that is less accurate